bundle/src/main/java/com/cognifide/cq/cqsm/core/progress/ProgressImpl.java

```diff
@@ -68,7 +66,7 @@ public void addEntry(Message message, Status status) {
 
 	@Override
 	public boolean isSuccess() {
-		calculateSuccess();
+		success = ProgressHelper.calculateSuccess(entries);

```

Comment:
Why you use field success? It's value is changed every time method isSuccess() is invoked.

Reply:
This field has been removed.